newsroom/push/views.py

```diff
@@ -65,7 +66,7 @@ async def handle_publish_text_item(_, item):
 
 async def handle_publish_planning_featured(_, item):
     assert item.get("_id"), {"_id": 1}
-    service = get_resource_service("agenda_featured")
+    service = FeaturedService()
```

Comment:
Async resources have a find_by_id for use cases like this, so you don't have to construct a query to get by id